### ISSUE TITLE
Fix Pa11y CI step connection errors

### ISSUE THREAD

**Issue by jtwillis92** [raft-tech/TANF-app#870]
**Description:**
Currently the Pa11y Accessibility Testing step is failing in Circle CI due to a connection issue. It looks like historically this has been passing with a [false positive success state](https://app.circleci.com/pipelines/github/HHS/TANF-app/497/workflows/63b7bdea-b41f-4374-ac10-71b1e5ed60eb/jobs/959/parallel-runs/0/steps/0-107) due to failures to download Chromium.

With https://github.com/raft-tech/TANF-app/pull/857 some changes were introduced that allow us to cache binaries downloaded by Yarn (like Chromium, Cypress, etc) so the step is now no longer failing to download Chromium and instead reveals the underlying issue:
```
[1] wait-on http://localhost:3000/ exited with code 0
--> Sending SIGTERM to other processes..
[0] yarn start:ci exited with code SIGTERM
$ /home/circleci/tdp-apps/tdrs-frontend/node_modules/.bin/pa11y-ci --config .pa11yci.json
Running Pa11y on 2 URLs:
 > http://localhost:3000 - Failed to run
 > http://localhost:3000/edit-profile - Failed to run

Errors in http://localhost:3000:

 • Error: net::ERR_CONNECTION_REFUSED at http://localhost:3000

Errors in http://localhost:3000/edit-profile:

 • Error: net::ERR_CONNECTION_REFUSED at http://localhost:3000/edit-profile
```

From the output we can see that the local app server is being prematurely exited before the pa11y tests run: 
```
[0] yarn start:ci exited with code SIGTERM
```

Comparing this script in `package.json` with the working `cy:run:ci` script we can see the difference that causes it to exit early:
```
"cy:run:ci": "concurrently -k -s first 'yarn start:ci' 'wait-on http://localhost:3000/ && yarn cy:run'",
```
vs
```
"test:accessibility": "concurrently -k -s first 'yarn start:ci' 'wait-on http://localhost:3000/' && yarn pa11y-ci --config .pa11yci.json",
```

Specifically:
```
'wait-on http://localhost:3000/ && yarn cy:run'
```
vs
```
'wait-on http://localhost:3000/' && yarn pa11y-ci --config .pa11yci.json
```

Due to the `pa11y-ci` command being invoked outside of the single quotation marks BASH will treat this as a separate clause and exits from the `yarn start:ci` command run at the beginning of the concurrently invocation.

Moving the `wait-on` command and `yarn pa11y-cli` into the same quotation block resolves the issue locally:
```
❯ concurrently -k -s first 'yarn start:ci' 'wait-on http://localhost:3000 && yarn pa11y-ci --config .pa11yci.json'                                                                                              ─╯
yarn run v1.22.10
$ CI=true HOST=localhost BROWSER=none yarn react-scripts start
$ /home/jwillis/Raft/Repositories/TANF-app/tdrs-frontend/node_modules/.bin/react-scripts start
[0] Attempting to bind to HOST environment variable: localhost
[0] If this was unintentional, check that you haven't mistakenly set it in your shell.
[0] Learn more here: https://cra.link/advanced-config
[0] 
[0] ℹ ｢wds｣: Project is running at http://localhost:3000/
[0] ℹ ｢wds｣: webpack output is served from 
[0] ℹ ｢wds｣: Content not from webpack is served from /home/jwillis/Raft/Repositories/TANF-app/tdrs-frontend/public
[0] ℹ ｢wds｣: 404s will fallback to /
[0] Starting the development server...
[0] 
[0] Compiled successfully!
[0] 
[0] You can now view tdrs-frontend in the browser.
[0] 
[0]   http://localhost:3000
[0] 
[0] Note that the development build is not optimized.
[0] To create a production build, use yarn build.
[0] 
yarn run v1.22.10
$ /home/jwillis/Raft/Repositories/TANF-app/tdrs-frontend/node_modules/.bin/pa11y-ci --config .pa11yci.json
[1] Running Pa11y on 2 URLs:
[1]  > http://localhost:3000/edit-profile - 0 errors (within threshold of 2)
[1]  > http://localhost:3000 - 0 errors (within threshold of 2)
[1] 
[1] ✔ 2/2 URLs passed
Done in 4.77s.
[1] wait-on http://localhost:3000 && yarn pa11y-ci --config .pa11yci.json exited with code 0
--> Sending SIGTERM to other processes..
[0] yarn start:ci exited with code 1
```

**Acceptance Criteria:** 
_Create a list of functional outcomes that must be achieved to complete this issue_

- [ ] Pa11y Accessibility Tests run in Circle CI
- [ ] Testing Checklist has been run and all tests pass
- [ ] README is updated, if necessary

**Tasks:**
_Create a list of granular, specific work items that must be completed to deliver the desired outcomes of this issue_ 

- [ ] Update package.json `test:accessbility` script to fix the quotation issue noted above
- [ ] Run Testing Checklist and confirm all tests pass

**Notes:**
_Add additional useful information, such as related issues and functionality that isn't covered by this specific issue, and other considerations that will be helpful for anyone reading this_ 

**Supporting Documentation:**
_Please include any relevant log snippets/files/screen shots_

**Open Questions:**
_Please include any questions or decisions that must be made before beginning work or to confidently call this issue complete_
